dao/src/main/java/org/thingsboard/server/dao/resource/BaseResourceService.java

```diff
@@ -103,17 +104,19 @@ public void deleteResource(TenantId tenantId, TbResourceId resourceId) {
     }
 
     @Override
-    public PageData<TbResourceInfo> findAllTenantResourcesByTenantId(TenantId tenantId, PageLink pageLink) {
+    public PageData<TbResourceInfo> findAllTenantResourcesByTenantId(TbResourceInfoFilter tbResourceInfoFilter, PageLink pageLink) {
```

Comment:
it is better to use simple variable name that is more readable (just ```filter``` instead of ```tbResourceInfoFilter```)